package.yaml

```diff
@@ -3,9 +3,10 @@ version:             2.0.0.0
 synopsis:            error tracking through rollbar.com
 homepage:            https://github.com/flipstone/rollbar-haskell
 license:             MIT
+license-file:        LICENSE
 author:              Flipstone Technology Partners
-maintainer:          development@flipstone.com
-copyright:           Daggerboard Inc. makers of docmunch.com, Azara Solutions Inc.
+maintainer:          maintainers@flipstone.com
+copyright:           2025 Flipstone Technology Partners
```

Comment:
I don't think we should remove Daggerboard and Azara as some portions of the library could still be attributed to them. Probably they should be in "author" too, but not in maintainer.